stories/organisms/components/filePicker/FilePicker.stories.mdx

```diff
@@ -97,6 +97,34 @@ It is a combination of the [FileInput](/docs/atoms-fileinput--overview) and the
 
 <ArgsTable story="Overview" />
 
+## Clear all
+
+The `clearAll` property allows the user to add a clickable text to delete all selected files.  
+This text appears automatically if at least two files have been selected.  
+By default the `clearAll` value is `true`.
```

Comment:
I didn't notice that in the `FilePicker` implementation, but IMHO it would be better to revert the behavior and initialize the `clearAll` property to `false`.
In the interests of seeking the richest possible User eXperience, it is preferable not to force a behavior but rather to bring about ways to modify it.

Reply:
> > > @JoeyDjr By the way, I missed a comment in my review, I think we should rename also the `clearAll` property into `showClearAll` because the purpose is completely different depending on the name used.
> > > `clearAll` refers to the action itself `showClearAll` refers to the display of the button
> > > What do you think @ccamel @lolottetheclash @ErikssonJoakim ?
> > 
> > 
> > I agree but IMO something like `hasClearAll` makes more sense
> 
> Or maybe `isClearable`?

I think we loose the "all" aspect with this name

I think `showClearAll` is the best, or `displayClearAll`